builds/jazz-build-module/aws-lambda-events-module.groovy

```diff
@@ -0,0 +1,87 @@
+#!groovy?
+import groovy.json.JsonSlurperClassic
+import groovy.json.JsonOutput
+import groovy.transform.Field
+
+/*
+* -- Lambda event notification module
+*/
+
+@Field def config_loader
+
+def initialize(configLoader){
+  config_loader = configLoader
+}
+
+
+def checkSQSAndAddLambdaTrigger(queueName, lambdaARN) {
+  try {
+    sh "aws sqs get-queue-url --queue-name $queueName --profile cloud-api --output json"
+    echo "Queue exists and have access"
+    addLambdaTriggerToSqsQueue(true, queueName, lambdaARN)
+  } catch (ex) {
+    addLambdaTriggerToSqsQueue(false, queueName, lambdaARN)
+  }
+}
+
+def addLambdaTriggerToSqsQueue(isExists, queue_name, lambdaARN){
+    if(isExists){
+      def isDefined = listEventSourceMapping(queue_name, lambdaARN)
+      if(!isDefined){
+        createEventSourceMapping(queue_name, lambdaARN)
+      }
+    }else{
+      createSqsQueue(queue_name, lambdaARN)
+    }
+}
+
+def createSqsQueue(queue_name, lambdaARN){
+  try{
+    sh "aws sqs create-queue --queue-name ${queue_name} --attributes '{\"VisibilityTimeout\": \"165\"}' --profile cloud-api --output json"
```

Comment:
why are you providing the timeout here?

Reply:
I am able to get the multiple lambda trigger working. Also the for the above question, by default the timeout period is 30. To increase that timeout period we have to provide the timeout period explicitly

Reply:
but how did you come up with that number. @devsatishm are we providing a visibility timeout?